scripting/togglemusic_dhook.sp

```diff
@@ -33,81 +27,89 @@ StringMap g_smCommon;
 StringMap g_smRecent;
 StringMap g_smVolume;
 
-public Plugin myinfo =
+public Plugin myinfo = 
 {
-	name = PLUGIN_NAME,
-	author = "Mitch & Agent Wesker",
-	description = "Allows clients to toggle ambient sounds played by the map",
-	version = PLUGIN_VERSION,
+	name = "Toggle Music", 
+	author = "Mitch & Agent Wesker (Playa Edit)", 
```

Comment:
It's not exactly good practice to put your name in the credits for this small of a change

Reply:
Remove it if you like or give me more Stuff to change :P